### PR TITLE
three  tests passed

### PR DESCRIPTION
**Pull request by alexander-bick**
After quite a lot of trial and error I managed to write three tests that a get a passed 😅

### CODE REVIEW

File: components/Form/index.test.js
Comment (Kheirah) [inline components/Form/index.test.js:1]: You need to run prettier for this file.

**Review by Kheirah — APPROVED** [review]
LGTM!

File: components/Form/index.test.js
Comment (alexander-bick) [reply components/Form/index.test.js:1]: True, I forgot that 😅

Comment (alexander-bick) [reply components/Form/index.test.js:1]: Am I right that the Minimum Requirements are fulfilled after implementing the correct design (375x667)?

**Pull request merged**